h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -141,6 +142,89 @@ public void computeCrossValidation() {
+* can cause missing submodels at the end of the submodel array.
+*
+*  CV1:
+*   +------+--------+--------+--------+--------+------+--------+--------+--------+--------+--------+--------+
+*   | 0, 1 | 0, 0.9 | 0, 0.8 | 0, 0.7 | 0, 0.6 | 1, 1 | 1, 0.9 | 1, 0.8 | 1, 0.7 | 1, 0.6 | 1, 0.5 | 1, 0.4 |
+*   +------+--------+--------+--------+--------+------+--------+--------+--------+--------+--------+--------+
+*  CV2:
+*   +------+--------+--------+--------+--------+--------+------+--------+--------+--------+
+*   | 0, 1 | 0, 0.9 | 0, 0.8 | 0, 0.7 | 0, 0.6 | 0, 0.5 | 1, 1 | 1, 0.9 | 1, 0.8 | 1, 0.7 |
+*   +------+--------+--------+--------+--------+--------+------+--------+--------+--------+
+*
+*    ||
+*    ||
+*   \  /
+*    \/
+*
+*  Aligned Submodels:
+*
+*  CV1:
+*  +------+--------+--------+--------+--------+--------+------+--------+--------+--------+--------+--------+--------+
+*  | 0, 1 | 0, 0.9 | 0, 0.8 | 0, 0.7 | 0, 0.6 |  NULL  | 1, 1 | 1, 0.9 | 1, 0.8 | 1, 0.7 | 1, 0.6 | 1, 0.5 | 1, 0.4 |
+*  +------+--------+--------+--------+--------+--------+------+--------+--------+--------+--------+--------+--------+
+*
+*  CV2:
+*  +------+--------+--------+--------+--------+--------+------+--------+--------+--------+--------+--------+--------+
+*  | 0, 1 | 0, 0.9 | 0, 0.8 | 0, 0.7 | 0, 0.6 | 0, 0.5 | 1, 1 | 1, 0.9 | 1, 0.8 | 1, 0.7 |  NULL  |  NULL  |  NULL  |
+*  +------+--------+--------+--------+--------+--------+------+--------+--------+--------+--------+--------+--------+
+*
+* */
+  private double[] alignSubModelsAcrossCVModels(ModelBuilder[] cvModelBuilders) {
+    // maximum index of alpha change across all the folds
+    int[] alphaChangePoints = new int[_parms._alpha.length + 1];
+    int[] alphaSubmodels = new int[_parms._alpha.length];
+    for (int i = 0; i < cvModelBuilders.length; ++i) {
+      GLM g = (GLM) cvModelBuilders[i];
+      Map<Double, List<Submodel>> submodels = Arrays.stream(g._model._output._submodels)
+              .collect(Collectors.groupingBy(sm -> sm.alpha_value));
+      for (int j = 0; j < _parms._alpha.length; j++) {
+        alphaSubmodels[j] = Math.max(alphaSubmodels[j],
+                submodels.containsKey(_parms._alpha[j]) ? submodels.get(_parms._alpha[j]).size() : 0);
+      }
+    }
+
+    for (int i = 0; i < _parms._alpha.length; i++) {
+      alphaChangePoints[i+1] = alphaChangePoints[i] + alphaSubmodels[i];
+    }
+
+    double[] lambdas = new double[alphaChangePoints[_parms._alpha.length]];
+    for (int i = 0; i < cvModelBuilders.length; ++i) {
+      GLM g = (GLM) cvModelBuilders[i];
+      Submodel[] alignedSubmodels = new Submodel[alphaChangePoints[_parms._alpha.length]];
+      Log.info(":::{"+i+"} creating alignedSubmodels (.length = " + alphaChangePoints[_parms._alpha.length] + "); #submodels = " + g._model._output._submodels.length + "; selected submodel = "+g._model._output._selected_submodel_idx);
```

Comment:
does this need to be info level with flashy prefix?
I understand it to be convenient during dev, but it also tends to remain and then default logs become noisy with flashier prefixes until logs end up with less information but more ">>>", "::::", "****", "###" ...